src/main/java/com/world/demo/worldmap/controller/CountryController.java

```diff
@@ -13,25 +13,20 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.CrossOrigin;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 
 /**
  *
  * @author Mohamed.Hamed
  */
 @CrossOrigin(origins = { "http://localhost:3000", "http://localhost:4200" })
 @RestController
-public class ApplicationController {
+public class CountryController {
 
-    @Autowired
-    CityService cityService;
     @Autowired
     CountryService countryService;
 
-    public ApplicationController(CountryService countryService) {
+    public CountryController(CountryService countryService) {
```

Comment:
you don't have to inject countryService throw the controller constructor @Autowired will take care of it for you